reference/sql_workarounds.rst

```diff
@@ -124,41 +124,74 @@ There is a workaround: you can replicate the local table to a single shard on ev
 This will create a table with a single shard (non-distributed), but will
 replicate that shard to every node in the cluster. Now Citus will accept a join query between *here* and *there*, and each worker will have all the information it needs to work efficiently.
 
+.. _window_func_workaround:
+
+Window Functions
+----------------
+
+Currently Citus does not have out-of-the-box support for window functions on queries involving distributed tables, but there is a straightforward workaround. Window functions will work on a distributed table if
+
+1. They are in a subquery and
+2. They :code:`partition by` the distribution column of a distributed table
```

Comment:
What part of the window function is that? Can we be a bit more specific by categorizing this class of window functions? (Window functions which include the partition-by clause?